src/calculator/routes.ts

```diff
@@ -19,6 +20,15 @@ calculatorRoutes.get('/twitter/user/:id', function(req, res, next) {
     })
 })
 
+calculatorRoutes.get('/scrapper/user-verified/:verified/user-joined/:joined', function(req, res, next) {
+  const userCredibility = scrapperUserCredibility(req.params.verified == 'true', Number(req.params.joined))
+  if (res.statusCode == 200){
+    res.send(userCredibility)
+  } else { 
+    next(new HttpError(res.statusCode))
+  } 
```

Comment:
There is no need for this check. On our architecture the controller only calls the service, nothing more.
Also you are comparing the `response` status code, which doesn't have sense because you don't receive a response, you _respond_ with a `response`.